database/dbutil.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/coreos/clair/pkg/commonerr"
 	"github.com/coreos/clair/pkg/pagination"
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
is this change also related?

Reply:
This change isn't related to the actual issue per se, it's just that my editor will not save this file without making this change and discussing it within my group we thought we'd leave it in as it helps with code correctness.
However, I am happy to remove it, just trying to get a feel for coding style etc.

Reply:
Just for reference the editor is just making it more explicit that the `mapset` package is actually provided by `golang-set` as it's not necessarily obvious from the name (from 2014: https://github.com/deckarep/golang-set/issues/11).  Personally having the declaration there makes the code feel a little more readable, but I can see that it might not want to be associated with this PR.